project_template/go.sh

```diff
@@ -8,22 +8,30 @@ EOF
 
 set -e
 
+# Detect if this script is run within an existing virtualenv and abort
+if [[ $VIRTUAL_ENV ]]; then
+    echo "ERROR: $0 will not work properly if run inside a venv; deactivate '`basename $VIRTUAL_ENV`'"
+    exit 1
+fi
```

Comment:
I just retried this and couldn't reproduce any problem. I vaguely remember it picking up the wrong python version etc, but that might have been due to other issues that are now fixed. So maybe it's not needed